### PR TITLE
Staging fixes11

### PR DESCRIPTION
**Pull request by Bod9001**
### Changelog

CL: [Improvement] dev menu can spawn stacks with x amount
CL: [Fix] fixes Getting cloned with default character sheet
CL: [Fix] Fixes jungle trees being Pushable
CL: [Fix] Fixes wearable  Speech modiffier getting stuck on character
CL: [Fix] Fixed an issue with UI actions not stacking properly

### CODE REVIEW

**Comment by corp-0** [comment]
fixes what cloning issue?

fixes what wearable speech modifier issue?

Just a reminder to be more verbose in the changelogs :B

There also seems to be a fix that isn't in the logs

File: UnityProject/Assets/Scripts/Clothing/WearableSpeechMod.cs [hunk 1]
```diff
@@ -43,12 +43,12 @@ void IItemInOutMovedPlayer.ChangingPlayer(Mind hideForPlayer, Mind showForPlayer
 	{
 		if (hideForPlayer != null)
 		{
-			hideForPlayer.inventorySpeechModifiers |= modifier;
+			hideForPlayer.inventorySpeechModifiers &= ~modifier;
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Clothing/WearableSpeechMod.cs:46]: what is ``~`` doing?

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Clothing/WearableSpeechMod.cs:46]: It inverts it looks like

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Clothing/WearableSpeechMod.cs:46]: idk I'm not good at binary operators

**Review by corp-0 — APPROVED** [review]

**Pull request merged**